config/testnet/exceptions.js

```diff
@@ -59,4 +61,7 @@ module.exports = {
 		'12298100805070303137', // 3057955 - Delegate transaction
 	],
 	transactionFee: [],
+	disableTransactions: {
+		dappTransfer: 5594491, // Disable Dapp Transfer on this block height
```

Comment:
Not sure about property name and comment here. We're not supporting dapp transfer transactions anymore, so from logical point of view - exceptions should `allow` those up to certain block height, not `disable` from certain height.

Reply:
Its the same thing :) 

Think in this direction. We allowed dapp transfer from start of the chain starting from that that particular block height we disabled those.